documentation/examples/prometheus.yml

```diff
@@ -27,4 +27,17 @@ scrape_configs:
     # scheme defaults to 'http'.
 
     static_configs:
-      - targets: ['localhost:9090']
+      - targets: 
+        - 'localhost:9090'
```

Comment:
sorry i can't reproduce it myself, was probably a problem with indentation. when another user asked about his job not being scraped, i jumped to the wrong conclusion.